app/plugins/prepopulate/prepopulatePlugin.php

```diff
@@ -525,15 +525,16 @@ public function prepopulateFields(&$t_instance, $pa_options=null) {
 		}
 
 
-		if(isset($_REQUEST['form_timestamp']) && ($_REQUEST['form_timestamp'] > 0)) { $_REQUEST['form_timestamp'] = time(); }
-		$t_instance->update(['force' => true, 'hooks' => false]);
-
-		if($t_instance->numErrors() > 0) {
-			foreach($t_instance->getErrors() as $vs_error) {
-				Debug::msg("[prepopulateFields()] there was an error while updating the record: ".$vs_error);
+		if ($t_instance->getChangedFieldValuesArray()) {
```

Comment:
Won't this always evaluate to true? getChangedFieldValuesArray returns an array, which will be empty (but array-is) if nothing has changed. Need to check size of array here?

Reply:
empty arrays are falseish:
```
 php -a
Interactive shell

php > if ([]) { echo 'true';} else { echo 'false';}
false
```
But happy to put a count around it

Reply:
My preference would be to test count explicitly

Reply:
Sorted

Reply:
Thank you!